src/utils/convert-utils.ts

```diff
@@ -0,0 +1,38 @@
+import { gql, request } from "graphql-request";
+import { Network } from "../types/network";
+
+const hpsmGraphEndpoint = "https://api.thegraph.com/subgraphs/name/handle-fi/handle-psm";
+
+type Peg = {
+  fxToken: string;
+  peggedToken: string;
+};
+
+export const getTokenPegs = async (): Promise<Peg[]> => {
+  const response = await request(
+    hpsmGraphEndpoint,
+    gql`
+      query {
+        pairs {
```

Comment:
This defaults to 100. Let's set it to 1k
```suggestion
        pairs(first: 1000) {
```